docs/data/data-grid/api-object/UseGridApiRef.js

```diff
@@ -0,0 +1,27 @@
+import * as React from 'react';
+import Button from '@mui/material/Button';
+import Stack from '@mui/material/Stack';
+import Box from '@mui/material/Box';
+import { DataGridPro, useGridApiRef } from '@mui/x-data-grid-pro';
+import { useDemoData } from '@mui/x-data-grid-generator';
+
+export default function UseGridApiRef() {
+  const { data } = useDemoData({
+    dataSet: 'Commodity',
+    rowLength: 100,
+    maxColumns: 6,
+  });
+
+  const apiRef = useGridApiRef();
+
+  const handleGoToPage1 = () => apiRef.current.setPage(1);
+
+  return (
+    <Stack spacing={2} sx={{ width: '100%' }} alignItems="flex-start">
```

Comment:
We have 1 in https://deploy-preview-5273--material-ui-x.netlify.app/x/react-data-grid/rows/#UpdateRowsProp.tsx and the other demos. Would this look better?

```suggestion
    <Stack spacing={1} sx={{ width: '100%' }} alignItems="flex-start">
```

Reply:
It's still 2.

<img width="250" src="https://user-images.githubusercontent.com/42154031/179772607-8fb203a2-42e1-4c7d-85b1-537d047f55ee.png" alt="image" style="max-width: 100%;">